### DISCUSSION TITLE
Seperate modded heists when buying a contract

### DISCUSSION THREAD

**Discussion by erenkarakal** [diesel-modding/PAYDAY-2-BeardLib#576] [Ideas]
**Is your feature request related to a problem? Please describe.**
It gets messy when custom contracts are mixed with vanilla ones.

**Describe the solution you'd like**
When buying a contract, seperate modded heists from the vanilla ones like this:
**vanilla**
bain
locke
...
**modded**
contractor1
contractor2
...

**Describe alternatives you've considered**
A button to hide all custom heists or switch between only custom / vanilla heists

**Comment by segabl** [comment]
https://modworkshop.net/mod/30027
Not quite what you asked for but solves the mess of too many custom contractors

**Comment by erenkarakal** [comment]
> https://modworkshop.net/mod/30027 Not quite what you asked for but solves the mess of too many custom contractors

thank you, i couldnt find it myself for some reason
